apps/learner/src/lib/components/Badge/Badge.svelte

```diff
@@ -3,7 +3,7 @@
   import type { HTMLAttributes } from 'svelte/elements';
 
   export interface Props extends HTMLAttributes<HTMLDivElement> {
-    variant: 'blue' | 'orange' | 'amber' | 'teal' | 'rose' | 'purple' | 'slate';
+    variant: 'blue' | 'orange' | 'amber' | 'teal' | 'rose' | 'purple' | 'slate' | 'darkSlate';
```

Comment:
Can use '-' instead. 
```suggestion
    variant: 'blue' | 'orange' | 'amber' | 'teal' | 'rose' | 'purple' | 'slate' | 'slate-dark';
```

You can consider using `'slate-light'` or keep it as `slate`.